daemon/algod/api/algod.oas2.json

```diff
@@ -1433,6 +1433,145 @@
         }
       ]
     },
+    "/v2/applications/{application-id}/boxes": {
```

Comment:
Should we expose `total-box-bytes` and `total-boxes` in the [Account definition](https://github.com/algorand/go-algorand/blob/4688f93ac9bbbb346a871f08602b1fce20ee3d58/daemon/algod/api/algod.oas2.json#L1938)?

Reply:
@jj @michaeldiamant  ^^^

Reply:
@tzaffi Thanks for calling it out - I opened https://github.com/algorand/go-algorand/issues/4478 to document the request.

Provided it works for you, I think we can resolve the thread and continue discussion out-of-band to the PR.